single-cell-portal/variables.tf

```diff
@@ -83,14 +83,14 @@ variable "enable_logging" {
 }
 variable "internal_range" {
```

Comment:
For staging (not production), we will also need to add a list of IP addresses so that Travis can connect to the Mongo instance via tcp:27017 (probably its own firewall rule).  I was able to pull the list from here: https://docs.travis-ci.com/user/ip-addresses/

Reply:
Yep, done!